### PR TITLE
Fix mashup.js undefined error

### PR DESCRIPTION
**Pull request by BlackWolfie**
Add a check on the variable 'source' to see if this variable is not undefined

### CODE REVIEW

File: _templates/mashup-0.3.5/mashup.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ var MashupTemplate = new Class({
       var source = item.get('src');
 
       berta.options.iframeWrapperWhiteList.each(function (whiteList) {
-        if (source.indexOf(whiteList) > -1) {
+        if (source !== 'undefined' && source.indexOf(whiteList) > -1) {
```
Comment (uldisrudzitis) [inline _templates/mashup-0.3.5/mashup.js]: This would be more readable, try if this works at all:
```
if (source && source.indexOf(whiteList) > -1) {
````

Comment (BlackWolfie) [reply _templates/mashup-0.3.5/mashup.js]: I tried it several times and didn't get an undefined variable error

Comment (uldisrudzitis) [reply _templates/mashup-0.3.5/mashup.js]: Can you debug and find how comes that source is undefined. It means that `iframe` is without `src` attribute? Can you find the reason?

Comment (vteivans) [reply _templates/mashup-0.3.5/mashup.js]: I agree, with @uldisrudzitis - this doesn't actually fix the issue, it just hides the error. We need to understand why is the source missing.

Comment (BlackWolfie) [reply _templates/mashup-0.3.5/mashup.js]: I did my research and the only time I can get an empty 'src' attribute from an iframe is the time it was added by the user without source.

**Pull request merged**